spec/unit/frameSpec.js

```diff
@@ -0,0 +1,45 @@
+describe("Frame", function () {
+  let frame;
+  
+  beforeEach(function () {
+    frame = new Frame();
+  });
+
+  describe("any new instances of thermostat class have empty rolls array", function () {
+    it("rolls array is empty for frame instance", function () {
+      expect(frame._rolls).toEqual([]);
+    });
+  });
+
+  describe("adds score after a bowl", function () {
+    it("adds score into rolls array", function () {
+      frame.addRoll(5);
+      expect(frame._rolls.length).toEqual(1);
+    });
+  });
+
+  describe("adds bonus score to bonus score variable", function () {
+    it("correctly updates bonus score", function () {
+      frame.addBonusScore(5);
+      expect(frame._bonus_score).toEqual(5);
+    });
+  });
```

Comment:
Don't test state. It's like private methods.
You should only test the public interface of your class, so that tests do not fix your code into a specific implementation.